OIPA/iati/parser/tests/test_2_03_parser.py

```diff
@@ -1265,3 +1265,49 @@ def test_activity_result_document_link(self):
                          dummy_document_link.file_format)
         self.assertEqual(document_link.activity, self.activity)
         self.assertEqual(document_link.result, self.result)
+
+
+class ResultDocumentLinkTitleTestCase(TestCase):
+
+    def setUp(self):
+        # 'Main' XML file for instantiating parser:
+        xml_file_attrs = {
+            "generated-datetime": datetime.datetime.now().isoformat(),
+            "version": '2.03',
+        }
+        self.iati_203_XML_file = E("iati-activities", **xml_file_attrs)
+
+        dummy_source = synchroniser_factory.DatasetFactory.create(
+            name="dataset-2"
+        )
+
+        self.parser_203 = ParseManager(
+            dataset=dummy_source,
+            root=self.iati_203_XML_file,
+        ).get_parser()
+
+        self.parser_203.default_lang = "en"
+
+        assert (isinstance(self.parser_203, Parser_203))
+
+        # Version
+        current_version = VersionFactory(code='2.03')
+
+        # Related objects:
+        self.activity = iati_factory.ActivityFactory.create(
+            iati_standard_version=current_version
+        )
+        self.document_link = iati_factory.DocumentLinkFactory. \
+            create(url='http://someuri.com')
+
+        self.parser_203.register_model('Activity', self.activity)
+        self.parser_203.register_model('DocumentLink', self.document_link)
+
+    def test_result_document_link_title(self):
+        self.parser_203 \
+            .iati_activities__iati_activity__result__document_link__title(None)
```

Comment:
Yea so let's pass an actual XML element here :) Yeah, it work's without doing this, but would make sense to do it as the purpose of the test is to test how the parser is dealing with XML element